services/cases-api/README.md

```diff
@@ -34,47 +34,114 @@ Request type: GET
 Headers
 Authorization: {user id}
 
-## Example json payload (POST)
+## Example json payload (PUT)
 
 ```
 {
-	"personalNumber": 198602011212,
-	"type": "VIVA_CASE",
-	"data": {
-		"id1": "value12",
-		"id2": "value23",
-		"id3": "value34",
-		"id4": [1323,3566,6443,2345,23321]
-	}
+  "status": {
+    "type": "active:ongoing",
+    "name": "Pågående",
+    "description": "Du har påbörjat en ansökan. Du kan öppna din ansökan och fortsätta där du slutade."
+  },
```

Comment:
A suggestion here is to only submit the statusType, then the lambda can retrieve the additional information saved on the status type. Unless we want to save the status information in the app and on AWS, this would result in two data sources that we need to update instead of one.
statusType: "active:ongoing"

Reply:
Done!